velox/dwio/parquet/RegisterParquetReader.cpp

```diff
@@ -14,8 +14,6 @@
  * limitations under the License.
  */
 
-#include "velox/dwio/parquet/RegisterParquetReader.h"
```

Comment:
how come we don't need to include the corresponding header file anymore?

Reply:
We don't actually need the header in the cpp file since it only has the declaration of the registration functions. 
The cpp file has the registration function definitions and we always link against the library the cpp file is part of.
The DWRF change has this pattern as well.

Reply:
To clarify, this is a cleanup.